project/LinkerdBuild.scala

```diff
@@ -12,7 +12,13 @@ object LinkerdBuild extends Base {
   val Bundle = config("bundle") extend Minimal
   val Dcos = config("dcos") extend Bundle
 
+  val configCore = projectDir("config")
+    .withTwitterLibs(Deps.finagle("core"))
+    .withLibs(Deps.jackson)
+    .withLib(Deps.jacksonYaml)
+
   val consul = projectDir("consul")
+    .dependsOn(configCore)
```

Comment:
If you will happen to have a better idea - please let me know, not huge fan of current situation as well. :/